src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -141,7 +91,7 @@ private static IEnumerable<ProjectContext> GetMatchingProjectContexts(IEnumerabl
         /// <param name="outputPath">Location of published files</param>
         /// <param name="configuration">Debug or Release</param>
         /// <returns>Return 0 if successful else return non-zero</returns>
-        private static int Publish(ProjectContext context, string outputPath, string configuration)
+        private static int PublishRidFxPermutation(ProjectContext context, string outputPath, string configuration)
```

Comment:
Not a huge fan of this function name. It's not immediately apparent that a context represents an RID->Fx Pair

`PublishContext`?